assignments/arrays.js

```diff
@@ -63,34 +63,74 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
-
+// console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+for(i = 0; i < inventory.length; i++) {
```

Comment:
I like your work, Sharde. For this problem, we don't need a for loop because we only want value. A for loop makes the computer do more work than it needs to.